src/alaskaStyle.scss

```diff
@@ -3,12 +3,21 @@
 
 // ---------------------------------------------------------------------
 
+// Support for fallback values
+@import "./../node_modules/@aurodesignsystem/design-tokens/dist/tokens/SCSSVariables.scss";
+
 // Support for auroElement styles
 @import "./../node_modules/@aurodesignsystem/webcorestylesheets/dist/auroElement/auroElement";
 
 :host {
   display: inline-block;
+  // listing both legacy and newer tokens to support
+  // transition to new naming spec.
+  //
+  // TO DO: Remove --auro token when confident
+  // that new spec has been adopted
   --auro-size-lg: 100%;
+  --ds-size-300: 100%;
```

Comment:
What are these two variables for? I do not see them being used in the code.

Reply:
See comment note

```
  // TO DO: Remove --auro token when confident
  // that new spec has been adopted
```

These are both there to support a fallback scenario until everyone has moved to the new `-ds` tokens.